### PR TITLE
Updated Cypress Test

### PR DESCRIPTION
**Pull request by Haz-git**

### CODE REVIEW

File: cypress/integration/search-workout-templates.spec.ts [hunk 1]
```diff
@@ -8,10 +8,22 @@ describe('Search workout templates', () => {
...
-    it('user can search for a workout template', () => {
+    it('can navigate to search page', () => {
         cy.visit('/');
         cy.findByRole('button', {
             name: /Search Workouts/i,
         }).click({ force: true });
     });
+
+    it('can search for a workout template', () => {
+        cy.get('input[data-cy="search-workout-input"]')
```
Comment (n8rzz) [inline cypress/integration/search-workout-templates.spec.ts:19]: be wary of using custom selectors to find elements. is cypress able to find by `role`? or find by `text`? or find by something a user would use to interact with the button?

Comment (Haz-git) [reply cypress/integration/search-workout-templates.spec.ts:19]: Interestingly enough, I think they recommend using selectors:
https://docs.cypress.io/guides/references/best-practices

Thoughts?

Comment (n8rzz) [reply cypress/integration/search-workout-templates.spec.ts:19]: So, kind of.  

In the old days, automated testing solutions used css class names, ids, or XPath. What they are getting at there by suggesting a `data-` attribute isn't wrong. It is useful to be able to grab elements via a `data` attribute. However, users don't use your app using data attributes. This is taking a page out of `testing-library`'s stance. https://testing-library.com/docs/guiding-principles.

TL;DR If you want to test your app the way users use it, you should test your app the way users use it. 😉

Comment (Haz-git) [reply cypress/integration/search-workout-templates.spec.ts:19]: That makes sense. I'll refactor that when I get a chance.
